site/docs/Community-Events.md

```diff
@@ -0,0 +1,55 @@
+#Upcoming Events
```

Comment:
I would maybe reformat this a bit to put a space in between the `#` and the text, to make the markdown easier to read. And perhaps add lines in between the headings.